src/test/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannelIT.java

```diff
@@ -29,9 +31,14 @@
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 
+@RunWith(Parameterized.class)
```

Comment:
Did you try to migrate to JUnit 5 before? It should be fairly easy and we could avoid this ugly syntax.

Reply:
It's done in JUnit 5. I must admin the previous version was quite handy for parameterizing the whole class :)